src/backends/unicodeplots.jl

```diff
@@ -184,13 +185,16 @@ function addUnicodeSeries!(
         end
     elseif st in (:surface, :wireframe)  # 3D
         colormap = get(series[:extra_kwargs], :colormap, :none)
```

Comment:
why do you need an extra keyword for that instead of using `get_colorgradient(series)` ?

Reply:
Because it would allow to use the colormaps defined in `UnicodePlots` which are not available in `Plots` (and which render better, because they are more suited to ansi 256 color codes (e.g. jet) on a terminal).